pkg/apis/build/v1alpha2/builder_types.go

```diff
@@ -55,6 +55,12 @@ type NamespacedBuilderSpec struct {
 	BackwardsCompatibleServiceAccount string `json:"serviceAccount,omitempty"`
 }
 
+// +k8s:openapi-gen=true
+type SignaturePath struct {
+	KeyName string `json:"keyName"`
+	Path    string `json:"path"`
+}
```

Comment:
If I'm understanding this correctly, a builder can be signed by multiple (cosign) secrets/keys. The purpose of this struct is to expose which keys were used and their corresponding signature (container image) locations in the status of the builder.

If so, maybe a better name for this would be
```suggestion
type CosignSignature struct {
	SigningSecret string `json:"signingSecret"`
	TargetImage    string `json:"targetImage"`
}
```

Reply:
Yes, that's correct. I'll rename it.

Reply:
I've renamed the field `Path` to `TargetDigest` as it's going to be holding the signature digest and not the image digest itself (that information is in another field of the `BuilderStatus` struct).

Reply:
Renamed.